.expeditor/end_to_end.pipeline.yml

```diff
@@ -370,6 +370,18 @@ steps:
             - BUILD_PKG_TARGET=x86_64-windows
             - BUILDKITE_AGENT_ACCESS_TOKEN
 
+  - label: "[:linux: test_pkg_bulkupload]"
+    command:
+      - .expeditor/scripts/end_to_end/setup_environment.sh dev
+      - test/end-to-end/test_pkg_bulkupload.sh
+    expeditor:
+      executor:
+        docker:
+          privileged: true
+          environment:
+            - ACCEPTANCE_HAB_AUTH_TOKEN
```

Comment:
You're going to need a `BUILD_PKG_TARGET` in here.

Reply:
Added in this commit https://github.com/habitat-sh/habitat/pull/7113/commits/e9f1516cefa6d66c5b1b5d082eb7c99ce97cbb95